app/controllers/admin/product_import_controller.rb

```diff
@@ -0,0 +1,69 @@
+require 'roo'
+
+class Admin::ProductImportController < Spree::Admin::BaseController
+
+  def import
+    unless params[:file]
+      redirect_to '/admin/product_import', :notice => 'File not found or could not be opened'
+      return
+    end
```

Comment:
Maybe we can move this in a `before_filter`, just to keep the controller's action as clean as possible. What do you think?